<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Epub电子书工厂</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resource/icon.png</normaloff>../resource/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,5">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="metaDataLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>书籍信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="cover">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>330</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>330</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>点击选择封面图片。</string>
              </property>
              <property name="statusTip">
               <string>EPUB书籍的封面，单击此处修改封面图片，图片格式文件必须使用JPG格式。</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>240</width>
                <height>320</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
             </font>
            </property>
            <property name="text">
             <string>书籍名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bookTitle">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="statusTip">
             <string>请输入电子书的书名。</string>
            </property>
            <property name="text">
             <string>未命名书籍</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
             </font>
            </property>
            <property name="text">
             <string>书籍作者：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bookAuthor">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="statusTip">
             <string>请输入电子书的作者，有多个作者时用逗号分隔。</string>
            </property>
            <property name="text">
             <string>未知作者</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="chapterMenuLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>章节目录：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="epub">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="statusTip">
           <string>章节目录列表，任意章节都允许创建子章节内容。</string>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="chapterContentLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>章节内容：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="chapterTitle">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="statusTip">
           <string>章节标题，修改此处内容可以同步修改章节标题。</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="chapterContent">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="statusTip">
           <string>章节内容，数据以HTML格式保存。</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
             </font>
            </property>
            <property name="text">
             <string>引用URL：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="chapterURL">
            <property name="statusTip">
             <string>当前章节内容引用的URL地址，当章节内容有相对链接时，会根据此地址转化为绝对链接。</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>50</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChapter">
    <property name="title">
     <string>章节(&amp;C)</string>
    </property>
    <addaction name="actionInsertSiblingChapter"/>
    <addaction name="actionInsertChildChapter"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveChapter"/>
    <addaction name="actionRemoveAllChapters"/>
    <addaction name="separator"/>
    <addaction name="actionImportChapter"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>电子书(&amp;F)</string>
    </property>
    <addaction name="actionSelectCover"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>其它(&amp;O)</string>
    </property>
    <addaction name="actionSetStyle"/>
    <addaction name="actionSetConfig"/>
    <addaction name="actionClearCache"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAboutThis"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChapter"/>
   <addaction name="menuOther"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImportFromPage">
   <property name="text">
    <string>从单独页面导入章节...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionImportChapter">
   <property name="text">
    <string>批量导入章节...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>../resource/icon.png</normaloff>../resource/icon.png</iconset>
   </property>
   <property name="text">
    <string>导出Epub...</string>
   </property>
   <property name="statusTip">
    <string>将当前内容导出保存为Epub电子书。</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出Epub电子书工厂。</string>
   </property>
  </action>
  <action name="actionAboutThis">
   <property name="text">
    <string>关于本软件...</string>
   </property>
  </action>
  <action name="actionRemoveChapter">
   <property name="text">
    <string>删除章节</string>
   </property>
   <property name="statusTip">
    <string>删除当前选择的章节内容。</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionInsertSiblingChapter">
   <property name="text">
    <string>插入新章节</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInsertChildChapter">
   <property name="text">
    <string>插入子章节</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSelectCover">
   <property name="text">
    <string>选择封面图片...</string>
   </property>
  </action>
  <action name="actionSetStyle">
   <property name="icon">
    <iconset>
     <normaloff>../resource/style.png</normaloff>../resource/style.png</iconset>
   </property>
   <property name="text">
    <string>修改样式表模板...</string>
   </property>
  </action>
  <action name="actionRemoveAllChapters">
   <property name="text">
    <string>删除所有章节</string>
   </property>
  </action>
  <action name="actionSetConfig">
   <property name="text">
    <string>参数设置...</string>
   </property>
   <property name="statusTip">
    <string>设置软件的各项参数。</string>
   </property>
  </action>
  <action name="actionClearCache">
   <property name="text">
    <string>清除下载缓存...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
